docs-src/cloud/users-invite.md

```diff
@@ -11,11 +11,14 @@ tags:
 ---
 
 :::caution
```

Comment:
hmm... why is this a "caution" ?

just seems like an "info" callout -

Reply:
It was originally a caution - changing seemed out of scope but open to changing.